src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -30,6 +30,10 @@
                                          )
 
 ACRES_PER_SQM = 0.000247105
+SQM_PER_HECTARE = 10000
```

Comment:
I wonder if we should use `HECTARE_PER_SQM` and multiply instead of the other way round. It would be more consistent with our previous approach of `ACRES_PER_SQM`